internal/controller/proxmoxcluster_controller.go

```diff
@@ -168,6 +168,14 @@ func (r *ProxmoxClusterReconciler) reconcileNormal(ctx context.Context, clusterS
 	// If the ProxmoxCluster doesn't have our finalizer, add it.
 	ctrlutil.AddFinalizer(clusterScope.ProxmoxCluster, infrav1alpha1.ClusterFinalizer)
 
+	cpe := clusterScope.ControlPlaneEndpoint()
+	switch {
+	case cpe.Host == "":
+		conditions.MarkFalse(clusterScope.ProxmoxCluster, infrav1alpha1.ProxmoxClusterReady, infrav1alpha1.MissingControlPlaneEndpointReason, clusterv1.ConditionSeverityWarning, "The ProxmoxCluster is missing or waiting for a ControlPlaneEndpoint host")
```

Comment:
This information is essentially only visible until IPAM is reconciled, which if IIRC doesn't take more than a few seconds. It would probably be good to also log and/or create an event for either of these.

Reply:
Thanks, just rebased and added the logs.

I skipped events generation since I don't know how they should be formatted since there aren't events generated in the provider.